packages/multiply/src/calculateMultParams.ts

```diff
@@ -185,22 +187,14 @@ function getCloseToDaiParams(
   skipFL: boolean;
 } {
   const _skipFL = false;
-  const maxCollNeeded = vaultInfo.currentDebt
-    .times(1.00001 /* to account for not up to date value here */)
```

Comment:
Could you expand on this?

Reply:
this is due to the fact that moment when transaction is mined pight be several blocks later than when transaction parameters where calculated and Maker Protocol accumulated new fees. 

same debtOffset can be found in current oasis-borrow and this is replacement for it

Reply:
Ok cool, looks good.